libbeat/common/jsontransform/jsonhelper.go

```diff
@@ -49,3 +49,7 @@ func WriteJSONKeys(event common.MapStr, keys map[string]interface{}, overwriteKe
 
 	}
 }
+
+func CreateJSONError(message string) common.MapStr {
```

Comment:
Does this have to be exported? Seems simple enough to replicate if we need it elsewhere, similar to the `debug` function calls.